<!--
  2 failure:  :disappointed_rel..., :microscope: Ther...
  0 warning: 
  3 messages
  
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  :microscope: There are library changes, but no changes to the unit tests. That's OK as long as you're refactoring existing code, but will require an admin to merge this PR. Please see <a href="https://github.com/miniman42/titanium_mobile/blob/TIMOB26399_ios11_category_bugfix/README.md#unit-tests">README.md#unit-tests</a> for docs on unit testing.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :thumbsup: Hey!, You deleted more code than you added. That's awesome!
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.5.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10356-JRGKKIRVLWTDLF3YGOYSFMOX35ZUKZYGRNSWBLT7PXDDIDW375FA
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

Running "checkFormat:ios" (checkFormat) task
Fatal error: Error: Formatting incorrect on "iphone/Classes/TiAppiOSUserNotificationCategoryProxy.m", proposed changes: <?xml version='1.0'?>
<replacements xml:space='preserve' incomplete_format='false'>
<replacement offset='3201' length='9'>&#10;          </replacement>
<replacement offset='3276' length='9'>&#10;          </replacement>
<replacement offset='3366' length='72'>&#10;                                                                         </replacement>
<replacement offset='3460' length='62'>&#10;                                                               </replacement>
<replacement offset='3557' length='50'>&#10;                                                   </replacement>
<replacement offset='3666' length='72'>&#10;                                                                         </replacement>
</replacements>

npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :tada: Another contribution from our awesome community member, miniman42! Thanks again for helping us make Titanium SDK better. :thumbsup:
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/iphone/Classes/TiAppiOSUserNotificationCategoryProxy.m
+++ b/iphone/Classes/TiAppiOSUserNotificationCategoryProxy.m
@@ -72,12 +72,12 @@
                                                                           options:options] retain];
         } else {
 #endif
-        // For iOS 11, use the "hiddenPreviewsBodyPlaceholder" constructor
-        _notificationCategory = [UNNotificationCategory categoryWithIdentifier:identifier
-                                                                       actions:defaultActions
-                                                             intentIdentifiers:intentIdentifiers
-                                                 hiddenPreviewsBodyPlaceholder:hiddenPreviewsBodyPlaceholder
-                                                                       options:options];
+          // For iOS 11, use the "hiddenPreviewsBodyPlaceholder" constructor
+          _notificationCategory = [UNNotificationCategory categoryWithIdentifier:identifier
+                                                                         actions:defaultActions
+                                                               intentIdentifiers:intentIdentifiers
+                                                   hiddenPreviewsBodyPlaceholder:hiddenPreviewsBodyPlaceholder
+                                                                         options:options];
 #if IS_XCODE_10
         }
 #endif

